vector/v.in.ogr/main.c

```diff
@@ -1175,13 +1175,17 @@ int main(int argc, char *argv[])
 			      Ogr_fieldname, OFTIntegerListlength);
 		}
 		else {
-		    G_warning(_("Column type (Ogr_ftype: %d) not supported (Ogr_fieldname: %s)"),
-			      Ogr_ftype, Ogr_fieldname);
-		    buf[0] = 0;
-		    col_info[i_out].type = G_store(buf);
-		}
-		G_free(Ogr_fieldname);
-	    }
+                /* handle columns of unsupported data type */
+                G_warning(_("Column <%s> is of unsupported data type (Ogr_ftype: %d)\n"
+                           "It is omitted from import\n"), Ogr_fieldname,
+                          Ogr_ftype);
+                buf[0] = 0;
+                col_info[i_out].type = G_store(buf);
```

Comment:
As @metzm mentioned, Line 1183 (`col_info[i_out].type = G_store(buf);`)  may be removed.

Reply:
L1182 should then be removed too, no?

Reply:
> L1182 should then be removed too, no?

That's right. Not needed.

Reply:
Thanks! PR updated in [cd67af7](https://github.com/OSGeo/grass/pull/2630/commits/cd67af7292f9f6edb05238616529042d53259a62) accordingly.